src/Cache/Engine/RedisEngine.php

```diff
@@ -218,7 +218,12 @@ public function clear($check)
         }
         $keys = $this->_Redis->getKeys($this->_config['prefix'] . '*');
 
-        return !in_array(false, $this->deleteMany($keys));
+        $result = [];
+        foreach ($keys as $key) {
+            $result[] = $this->_Redis->delete($key) > 0;
```

Comment:
because `delete()` is to create the Multiple Keys command

e.g.

---

php
```
$keys = ['KEY1', 'KEY2', 'KEY3'];
$this->_Redis->delete($keys)
```
redis-cli
```
del KEY1 KEY2 KEY3
```
---
php
```
$keys = ['KEY1', 'KEY2', 'KEY3'];
foreach ($keys as $key) {
    $this->_Redis->delete($key);
}
```
redis-cli
```
del KEY1
del KEY2
del KEY3
```